docs/source/models/causes/neonatal/index.rst

```diff
@@ -349,7 +349,26 @@ Data Tables
       - GBD + assumption about relative risks + intervention model effects
       - see subcause models for details
 
+**Details of the** :math:`\text{PAF}_\text{LBWSG}` **calculation:**
 
+As stated in the table above, :math:`\text{PAF}_\text{LBWSG}` is the population attributable fraction of all-cause mortality for low birth weight and short gestation. It is computed so that PAF = 1 - 1 / E(:math:`\text{RR}_{\text{BW},\text{GA}}`) from the capped interpolated relative risk function (with expectation taken over the distribution of LBWSG exposure). 
+
+For the early neonatal age group, the LBWSG exposure at birth is used. For the late neonatal age group, we will use the LBWSG exposure at 8 days of life (start of the late neonatal age group after all early neonatal deaths have occurred). This LBWSG exposure is not directly available from GBD. Therefore, we will need to produce it ourselves according to the following steps:
```

Comment:
This is technically inaccurate for what is happening in the sim. The LNN mortality is happening at 14 days (7 + (28 - 7)) / 2. I dont think that would have a substantial impact but wanted to make sure that was clear.